Scripts/tests/emme_only/test_assignment.py

```diff
@@ -135,41 +135,17 @@ def test_assignment(self):
                         mtx[ass_class] = cost_data
 
     def test_transit_cost(self):
-        fares = pandas.DataFrame({
-            0: {
-                "firstb_single": 2.0,
-                "dist_single": 0.1
-            },
-            1: {
-                "firstb_single": 3.0,
-                "dist_single": 0.2
-            },
-            2: {
-                "firstb_single": 5.0,
-                "dist_single": 0.1
-            },
-            3: {
-                "firstb_single": 70.0,
-                "dist_single": 0.3
-            },
-            4: {
-                "firstb_single": 0.0,
-                "dist_single": 0.1
-            }
-        })
+        firstb_single = (2, 3, 5, 70, 0, 1.5)
+        dist_single = (0.1, 0.2, 0.1, 0.3, 0.1, 0.2)
+        fares = pandas.DataFrame(
+            {i: {"firstb_single": firstb_single[i],
+                 "dist_single": dist_single[i]}
+             for i in range(0, len(firstb_single))})
```

Comment:
Suggesting this slightly cleaner look on creating fare df. Made same change to test assignment in unit tests.